<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowBuy</class>
 <widget class="QDialog" name="MainWindowBuy">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>900</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>750</width>
    <height>900</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Диалог</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>//mac/Home/Downloads/fff8406e7b47fcdfea71e0e12b74b0ec.png</normaloff>//mac/Home/Downloads/fff8406e7b47fcdfea71e0e12b74b0ec.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Ваше имя:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditBuyerName"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>Купить</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="tableWidgetConcerts">
     <column>
      <property name="text">
       <string>Исполнитель</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Дата</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Цена</string>
      </property>
     </column>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
